geode-core/src/test/java/org/apache/geode/internal/net/SocketCreatorJUnitTest.java

```diff
@@ -120,28 +117,14 @@ public void configureSSLEngine() {
     SSLEngine engine = mock(SSLEngine.class);
     when(engine.getSSLParameters()).thenReturn(parameters);
 
-    final Object[] setProtocols = new Object[1];
-    doAnswer((Answer<Void>) invocation -> {
-      setProtocols[0] = invocation.getArgument(0);
-      return null;
-    }).when(engine).setEnabledProtocols(any(String[].class));
-
-    final Object[] setCiphers = new Object[1];
-    doAnswer((Answer<Void>) invocation -> {
-      setCiphers[0] = invocation.getArgument(0);
-      return null;
-    }).when(engine).setEnabledCipherSuites(any(String[].class));
+    ArgumentCaptor<String[]> engineArgumentCaptor = ArgumentCaptor.forClass(String[].class);
 
     socketCreator.configureSSLEngine(engine, "somehost", 12345, true);
 
     verify(engine).setUseClientMode(isA(Boolean.class));
     verify(engine).setSSLParameters(parameters);
-    verify(engine).setEnabledCipherSuites(isA(String[].class));
-    assertThat(setProtocols[0]).isNotNull();
-    String[] protocolStrings = (String[]) setProtocols[0];
-    assertThat(protocolStrings[0].equals("someProtocol"));
-    String[] cipherStrings = (String[]) setCiphers[0];
-    assertThat(cipherStrings[0].equals("someCipher"));
+    verify(engine).setEnabledProtocols(engineArgumentCaptor.capture());
+    verify(engine).setEnabledCipherSuites(engineArgumentCaptor.capture());
```

Comment:
You will want separate captors for each of these and then assert the values.
```java
ArgumentCaptor<String[]> enabledProtocols = ArgumentCaptor.forClass(String[].class);
verify(engine).setEnabledProtocols(enabledProtocols.capture());
assertThat(enabledProtocols.getValue()).containsExactly("someProtocol");

ArgumentCaptor<String[]> enabledCipherSuites = ArgumentCaptor.forClass(String[].class);
verify(engine). setEnabledCipherSuites(enabledCipherSuites.capture());
assertThat(enabledCipherSuites.getValue()).containsExactly("someCipher");
```

Reply:
The captor in this commit does capture both String[] values though.  The first capture() returns the String[] with "someProtocol" and the second returns the String[] with "someCipher".  I'll add the content checks.